lib/py/src/ext/protocol.tcc

```diff
@@ -431,6 +433,7 @@ bool ProtocolBase<Impl>::encodeValue(PyObject* value, TType type, PyObject* type
     }
```

Comment:
It shouldn't be...
The value is either null-checked or passed to `Py_INCREF` that does not accept null.
The intention here is to simply reject too long string values.
If you experienced null value here, we'll need to investigate on that matter too.